packages/webapp/src/containers/Finances/NewExpense/ExpenseCategories/index.jsx

```diff
@@ -13,10 +14,13 @@ import SeedImg from '../../../../assets/images/log/seeding.svg';
 import OtherImg from '../../../../assets/images/log/other.svg';
 import LandImg from '../../../../assets/images/log/land.svg';
 import { setSelectedExpenseTypes } from '../../actions';
+import { IconLink } from '../../../../components/Typography';
 import history from '../../../../history';
 import { withTranslation } from 'react-i18next';
 import { Grid } from '@mui/material';
 import PropTypes from 'prop-types';
+import { BsGear } from 'react-icons/bs';
```

Comment:
Just heads up, I will replace the icon with SVG in the [mockup](https://www.figma.com/file/96NZ02oYe3jpet1roUp0s0/Mockups?type=design&node-id=2148-38928&mode=design&t=N0mxQQIsFmzLgNSS-0) later in my PR for [LF-3553](https://lite-farm.atlassian.net/browse/LF-3553).

You mentioned in the ticket, but I will make some amount of changes on this component...
I should have done it before you started working on this, sorry!

Reply:
That's alright, no worries! I added the bare minimum so that the spotlight would make sense, feel free to make any changes needed.